deploy/addon/postgresql/pgpool.md

```diff
@@ -0,0 +1,347 @@
+
+Once replication is in place, you can use the `CC_PGPOOL_FOLLOWERS` environment variable to add the follower(s) to your Pgpool-II configuration. This variable is in **JSON** format, and must contain the **host**, **port** and **weight** of each follower.
+
+An example of the ``CC_PGPOOL_FOLLOWERS`` variable with two followers: 
+```json
+[
+  {
+    "hostname": "<DATABASE>-postgresql.services.clever-cloud.com",
+    "port": "<PORT>",
+    "weight": "1"
+  },
+  {
+    "hostname": "<DATABASE>-postgresql.services.clever-cloud.com",
+    "port": "<PORT>",
+    "weight": "1"
+  }
+]
+```
+
+The weight is used to specify the load balance ratio of the backends, you can also configure the weight of the leader with the `CC_PGPOOL_LEADER_WEIGHT` variable.
+
+{{< alert "info" "Information:" >}}
+You can set a higher value for followers to avoid overloading the leader with **SELECT** queries that could be made on followers.
+{{< /alert >}}
+
+Many other **load balancing** options are configurable with environment variables. You can refer to our [environment variables reference page](https://www.clever-cloud.com/doc/reference/reference-environment-variables/) and the [official Pgpool-II documentation](https://www.pgpool.net/docs/latest/en/html/runtime-config-load-balancing.html) for more information.
+
+### Health check
+
+Pgpool-II periodically connects to the configured PostgreSQL backends to detect any error on the servers or networks. If an error is detected, Pgpool-II performs failover or degeneration depending on the configurations. 
+
+The health check is not activated by default, but you can activate it with the `CC_PGPOOL_HEALTH_CHECK_PERIOD` variable.
+
+Many other health check options are configurable with environment variables, such as:
+
+`CC_PGPOOL_HEALTH_CHECK_TIMEOUT`: Specifies the timeout in seconds to give up connecting to the backend PostgreSQL if the TCP connect does not succeed within this time.  
+`CC_PGPOOL_HEALTH_CHECK_MAX_RETRIES`: Specifies the maximum number of retries to do before giving up and initiating failover when health check fails.  
+`CC_PGPOOL_HEALTH_CHECK_RETRY_DELAY`: Specifies the amount of time in seconds to sleep between failed health check retries.  
+`CC_PGPOOL_CONNECT_TIMEOUT`: Specifies the amount of time in milliseconds before giving up connecting to backend using connect() system call.
+
+You can consult the [official Pgpool-II documentation](https://www.pgpool.net/docs/latest/en/html/runtime-config-health-check.html) to get more information about the **health check**.
+
+### In memory cache
+
+In memory cache saves the pair of **SELECT** statement and its result. If the same **SELECTs** comes in, Pgpool-II returns the value from cache. This feature can be activated with the `CC_PGPOOL_MEMORY_CACHE_ENABLED=on` variable.
+
+Many other in memory cache options are configurable with environment variables, such as:
+
+`CC_PGPOOL_MEMQCACHE_TOTAL_SIZE`: Specifies the shared memory cache size in bytes.  
```

Comment:
I think you should prefix all of those items with `-` to make a list. I'm otherwise afraid that it will render on one line